examples/encrypting.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::Result;
+use ironoxide::prelude::*;
+use std::{fs::File, path::PathBuf};
+
+/// This is a very basic example of encrypting to a user_id and a group_id.
+/// The group that's encrypted to is created each run, but the user is just the current user.
+#[tokio::main]
+async fn main() -> Result<()> {
+    let (device_context, sdk) =
+        initialize_sdk_from_file(&PathBuf::from("examples/example-ironoxide-user.json")).await?;
+    encrypt_to_group(&sdk).await?;
+    encrypt_to_user(&sdk, device_context.account_id()).await?;
+    Ok(())
+}
+/// Create a group, asking the SDK to generate a unique ID for it.
+async fn encrypt_to_group(sdk: &IronOxide) -> Result<DocumentId> {
+    // start-snippet{encryptToGroup}
+    let group_id = create_group(sdk).await?;
+    let message = "This is my secret which a whole group should see.";
+    let encrypted_result = sdk
+        .document_encrypt(
+            message.as_bytes(),
+            &DocumentEncryptOpts::with_explicit_grants(None, None, true, vec![(&group_id).into()]),
```

Comment:
An alternative would be to `group_id.clone()` here

Reply:
I would still need the into(), so I'll stick with the borrow.